src/utils/spo.ts

```diff
@@ -69,6 +69,18 @@ export interface GraphFileDetails {
   VroomItemID: string;
 }
 
+export interface TenantSites {
+  Row: any[];
```

Comment:
Would it be helpful if we type this?

Reply:
What do you mean? I'm afraid I don't really understand what you are asking here

Reply:
I mean add a typescript type, instead of `any`...